client/src/ui/organisms/article-actions-container/index.scss

```
@@ -66,3 +66,9 @@
     }
   }
 }
+
+@media screen and (max-width: $screen-md) {
+  :root {
+    --sticky-header-height: 2rem;
+  }
+}
```

As this is only needed for mobile, can we just have the following?

```suggestion
/* ensure the breadcrumbs bar does not overlap the currently focused heading */
@media screen and (max-width: $screen-md) {
  :target {
    scroll-margin-top: 2rem;
  }
}
```

---

My rationale for adding this variable was to prepare for 

- making the actions header sticky on larger screens as well,
- making the BCD table header sticky as well (it will need to use the same offset from the top),
- making it possible to dynamically change this variable via a `ResizeObserver` if the header height should change dynamically.

But you're right, I shouldn't have done this according to YAGNI.

---

No, that all makes sense. Providing this additional context helps me understand you reasoning for this. I _was_ wondering whether there was a bigger context I was missing. Thank you for detailing your thought process.